test/integration/lib/bulk_download_service_test.rb

```diff
@@ -4,12 +4,24 @@
 
 class BulkDownloadServiceTest < ActiveSupport::TestCase
   include Minitest::Hooks
+  include SelfCleaningSuite
   include TestInstrumentor
```

Comment:
These three includes seem present in most, if not all, tests in this PR.

If feasible, I think abstracting them to a share file and perhaps importing that one thing here (and the 15+ equivalent other places) would be more maintainable than importing these three things.

Reply:
See above re: require/include precedence.  I'm not aware of a way to do this that won't result in errors.  Doesn't mean it isn't possible, I'm just not sure how it could be done.

Reply:
@eweitz actually I take that back - I think I know a way how to do it.  Will need to run some tests but I think I can do includes in helpers if I do it inside class definitions for the base classes of each test (`ActiveSupport::TestCase`, `ActionDispatch::IntegrationTest`, and `ActionController::TestCase`).